SoftLayer/CLI/autoscale/create.py

```diff
@@ -0,0 +1,124 @@
+"""Order/Create a scale group."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+from SoftLayer import utils
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import exceptions
+from SoftLayer.CLI import formatting
+from SoftLayer.CLI import helpers
+from SoftLayer.managers.autoscale import AutoScaleManager
+
+
+@click.command()
+@click.option('--name', help="Scale group's name.")
```

Comment:
![image](https://user-images.githubusercontent.com/78806346/165847224-dcf18b8d-60ae-4a55-bcf4-7c6c0b7b28ce.png)
Please add `required=True, prompt=True` to ensure the data input